### DISCUSSION TITLE
Admin event expiration

### DISCUSSION THREAD

**Discussion by jbman** [keycloak/keycloak#10646] [Ideas]
For Keycloak login events it is possible to have an expiration defined. But for admin events there is no expiration option. Is this by purpose? Anyone interested in an admin event expiration setting?

**Comment by jbman** [comment]
Additional remark: As the The setting is named "eventsExpiration" I initially assumed this would be applied to admin events, too.

**Comment by danielFesenmeyer** [comment]
Has been implemented in context of this issue: https://github.com/keycloak/keycloak/issues/12476
